packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -42,6 +43,13 @@ export interface ChatCompositeProps extends BaseCompositeProps<ChatCompositeIcon
  * @public
  */
 export type ChatCompositeOptions = {
+  /**
+   * Choose to use the composite form optimized for use on a mobile device.
+   * @remarks This is currently only optimized for Portrait mode on mobile devices and does not support landscape.
+   * @defaultValue false
+   * @alpha
+   */
+  mobileView?: boolean;
```

Comment:
We were looking to avoid the mobileView flag on chat composite because we don't have any features that present themself if it is set (i.e. if a developer sees this flag it confusing to set it to not get any different experience).
Currently the chat composite components should be responsive, is there a reason to have a different minWidth on desktop vs mobile?

Reply:
Oh I see its due to the participant pane? We can ignore that in the calculations as its being disabled by default and marked beta for GA.
cc @alkwa-msft from our discussion on this recently

Reply:
the participant pane, but if we don't count this pane, I can go with the same width of 19.5rem (just be aware of pane having scrollbar then when opened)

Reply:
Oh I see its due to the participant pane? We can ignore that in the calculations as its being disabled by default and marked beta for GA.
cc @alkwa-msft from our discussion on this recently